draft-js-dnd-plugin/src/Image/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import Draggable from '../components/block-draggable-wrapper';
+import Alignment from '../components/block-alignment-wrapper';
+
+class Image extends Component {
+  remove = (event) => {
+    event.preventDefault();
+    event.stopPropagation();
+
+    this.props.blockProps.onRemove(this.props.block.getKey());
+  };
+
+  alignLeft = () => {
+    this.props.align('left');
+  };
+
+  alignCenter = () => {
+    this.props.align('center');
+  };
+
+  alignRight = () => {
+    this.props.align('right');
+  };
+
+  render() {
+    const { blockProps, block, attachButtons, theme, alignment, onDragStart, draggable } = this.props;
+
+    const buttons = [
+      <span className={ theme.get('imageButton') }
+        onClick={this.alignLeft}
+        style={{ marginLeft: '-2.4em' }}
```

Comment:
You're right, but I've planned to get rid of these buttons anyways and replace them by a proper toolbar, so I didn't put a lot of effort in that. I'll just leave the styling until I'm done with the toolbar, which I will be able to work on and finish by the weekend.